src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -2688,6 +2738,15 @@ internal static extern
             [In] IntPtr lpPassword
         );
 
+
+        [DllImport(PinvokeDllNames.SetServiceObjectSecurityDllName, CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern
+        bool SetServiceObjectSecurity(
```

Comment:
Please add 
`[return: MarshalAs(UnmanagedType.Bool)]`
after line 2742.
(It is default but explicit is better)

Reply:
Done